DbgProvider/public/Debugger/DbgEngDebugger.cs

```diff
@@ -3189,7 +3304,7 @@ private void _EnsureSymbolsLoaded( DbgModuleInfo mod, CancellationToken cancelTo
                     if( mod.SymbolType == DEBUG_SYMTYPE.DEFERRED )
                     {
                         // TODO: progress output?
-                        CheckHr( m_debugSymbols.ReloadWide( " /f " + modFileName ) );
+                        CheckHr( DbgHelp.SymLoadModule( m_debugClient, mod.BaseAddress ) );
```

Comment:
>SymLoadModule [](start = 41, length = 13)

I'm curious: why this change?

DbgShell bypasses dbgeng and goes to dbghelp to query type information, because we simply can't get enough type info from dbgeng. But in this case, I'm concerned that dbgeng may be maintaining some state, and by bypassing it and going directly to dbghelp to load a PDB, now dbgeng's state might be wrong.

Reply:
Something had to be changed because `.reload /f ntdll.dll` doesn't tell DbgEng which ntdll.dll to load symbols for. I picked called DbgHelp directly over adding the base address & size to the reload call because the reload /f also causes DbgEng to send us ChangeSymbolState callbacks with no module for the argument.

DbgEng still knows about this happening - it's registered callbacks with DbgHelp, and we get a ChangeSymbolState callback from DbgEng before SymLoadModule has returned (and it actually passes along the right module base address, so we don't have to throw everything out!)